### DISCUSSION TITLE
Inaccurate coast line

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#550] [Q&A] [answered]
I am using OAM maps and the Elevate theme. Using the map France South I have recognized that the Coast line (blue color of the sea) sometimes does not follow exactly the coast. As an example a screenshot of Monaco. Is it a problem of the map data or of Cruiser's way to display the data? It happens with several maps like Germany North... It is zoom-independed. The rendering setting is on "quality". 
![Screenshot_2024-05-25-19-02-17-08_beed72cb86d752c6022398a83c94aee3](https://github.com/devemux86/cruiser/assets/138766725/1f3cc846-0321-4bf4-afe0-a9b5a0c1155d)

**Comment by devemux86** [comment]
@kds112 thanks for the report.

Which maps are loaded? Is also the world map loaded?

**Reply by kds112** [reply]
Several other OAM maps are loaded but not activated (10 maps of European contries). World map is loaded and activated.

**Reply by kds112** [reply, marked as answer]
Hey, it was a good hint... Deactivating the world map disappears the problem.
You mentioned in the past to "update" the world map? Maybe it should stay in the background when a map is activated?

**Reply by kds112** [reply]
And maybe showing in high zoom levels areas of downloaded but not activated maps? 🙃

**Reply by devemux86** [reply]
So the world map interferes here, it has lower quality because of the small size.

I plan to improve how it is used with the other maps.
